subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/PmdPlugin.java

```diff
@@ -61,7 +61,7 @@
  */
 public abstract class PmdPlugin extends AbstractCodeQualityPlugin<Pmd> {
 
-    public static final String DEFAULT_PMD_VERSION = "6.48.0";
+    public static final String DEFAULT_PMD_VERSION = "6.55.0";
```

Comment:
Can you add a note here that we also have to update PmdExtension links?

Reply:
Done.